ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -201,6 +382,85 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-workers-rhel8
+- as: azure-mag-ipi-fips-f28
```

Comment:
We have other jobs already covered `azure-mag-ipi` and `fips`, so we can remove this one